node/impl/client/client.go

```diff
@@ -399,7 +438,7 @@ func (a *API) ClientRetrieve(ctx context.Context, order api.RetrievalOrder, ref
 		order.Total,
 		order.MinerPeerID,
 		order.Client,
-		order.Miner)
+		order.Miner) // TODO: pass the store here   somehow
```

Comment:
do we want to complete this TODO?

Reply:
Nope, depends on a bunch of markets work